layout/layout.h

```diff
@@ -171,7 +171,17 @@ enum {
 	M0_LAYOUT_ENUM_TYPE_MAX = 32
 };
 
-enum { M0_DEFAULT_LAYOUT_ID = 1 };
+#ifdef CONFIG_X86_64
+	enum { M0_DEFAULT_LAYOUT_ID = 1 };
+#elif defined CONFIG_AARCH64
+	enum { M0_DEFAULT_LAYOUT_ID = 5 }; 
+/**
+ * Pages size on aarch64 Centos 7.9 : 64KB
+ * which corresponds to layout Id of 5.
+ */
```

Comment:
Elaborate it please like this:

```
/*
 * Layout id defines the object unit size which cannot be less than the page size:
 * 1 - 4KB, 2 - 8KB, 3 - 16KB and so on. The page size on RHEL aarch64 platform is
 * 64KB, hence the default layout id should be 5 (64KB).
 * /
```

And put it before `enum { M0_DEFAULT_LAYOUT_ID = 5 };`.

Reply:
fixed